pkg/trace/info/stats.go

```diff
@@ -288,6 +284,49 @@ func (s *SpansMalformed) String() string {
 	return mapToString(s.tagValues())
 }
 
+const (
+	maxAbsSamplingPriority = 10
```

Comment:
I'm a really big fan of the Go idiom of keeping variables as short as possible. We've been trying to follow that style throughout. Granted, this repository is a mix of both old and new and it's hard to know what to follow as a newcomer. You'll have to forgive my nitty comments. I should probably move the [styleguide](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines) that we have here too.

```suggestion
    // maxAbsPriority specifies the absolute maximum priority. For example, with a value
    // of 10, the priority range will be [-10, 10].
	maxAbsPriority = 10
```

I think it's fine to drop the word "sampling" here right?

Reply:
Hi Gabriel,

I agree in general with having short names, so far as they remain meaningful and easy to read, I will update it.

Thanks for the comments, there is nothing to forgive :D I thank them, I need your comments to get adjusted to the coding style and standards. 

Thank you!